src/app/user/common/user.service.ts

```diff
@@ -15,13 +15,14 @@ export class UserService {
   private headers = new Headers({'Content-Type': 'application/json'});
 
   constructor(private http: Http, private jsonp: Jsonp) { }
-
-  // only active users are retrieved
-  getUsers(): Observable<User[]> {
-    const url = userUrl + '/active';
+  // Page data always starts at offset 0 
+  // Only active users are retrieved
+  // Returns a JSON object with the data array of Users and totalItems count
+  getUsers(page: number): Observable<any> {
+    const url = userUrl + '/active?page=' + (page - 1) + '&size=10' + '&sort=id,asc';
     return this.http
                .get(url)
-               .map(res => res.json())
+               .map(res => JSON.parse(`{"data": ${JSON.stringify(res.json().content)}, "totalItems": ${res.json().totalElements} }`))  
```

Comment:
i think what you're trying to do is return an object, right? it's tricky with arrow functions. you need to wrap the object in parentheses, like

```
.map( res => ({data: res.json().content, totalItems: res.json().totalElements})
```

although you could also cache the results of `res.json()` and use a regular function

```
.map( res => { const data = res.json(); return {data: data.content, totalItems: data.totalElements}; })
```